src/manifest/pythonEnvironment.ts

```diff
@@ -33,12 +41,20 @@ export class PythonEnvironment implements Disposable {
 
   public get pythonPath() {
     return (
-      this.getPythonPathFromConfig() || this.executionDetails!.getPythonPath()
+      getResolvedConfigValue(
+        "dbtPythonPathOverride",
+        this.environmentVariables,
+      ) || this.executionDetails!.getPythonPath()
     );
   }
 
-  public get environmentVariables() {
-    return this.executionDetails!.getEnvVars();
+  public get environmentVariables(): EnvironmentVariables {
+    const envVarsDebug = this.executionDetails!.getEnvVars();
```

Comment:
Why Debug? Seems weird name for something we use to get the env vars for production purposes.